### ISSUE TITLE
Update pager heading levels in CMS

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#16077]
## Description
After completing the work in #16059, it was determined that all the pagers should have a heading level of H2, regardless of which view it's on.

## Acceptance Criteria
- [x] All pagers have a heading level of H2
- [x] Requires accessibility review

## Team
Please check the team(s) that will do this work.

- [x] CMS Team
- [ ] Public Websites
- [ ] Facilities
- [ ] Accelerated Publishing
